omod/src/main/java/org/openmrs/module/metadatasharing/web/controller/PublishController.java

```diff
@@ -49,13 +48,13 @@
  * The controller for publish related pages.
  */
 @Controller(MetadataSharingConsts.MODULE_ID + ".PublishController")
-public class PublishController {
+public class PublishController<GetLatestSubscriptionHeaderView> {
 	
 	/**
 	 * The view which returns XML serialized "header" variable from the model
 	 */
 	@Autowired
-	private GetSubscriptionHeaderView subscriptionHeaderView;
+	private GetLatestSubscriptionHeaderView subscriptionHeaderView;
```

Comment:
Does this class exist?

Reply:
This was suggested by @djazayeri on the ticket description